app/feature/feature-insurances/src/test/kotlin/com/hedvig/android/feature/insurances/data/GetInsuranceContractsUseCaseImplTest.kt

```diff
@@ -116,7 +118,12 @@ class GetInsuranceContractsUseCaseImplTest {
       )
       val result = subjectUseCase.invoke().first()
       assertk.assertThat(result).isRight().transform {
-        it.first().supportsTierChange
+        val first = it.first()
+        if (first is AbstractInsuranceContract.InsuranceContract) {
```

Comment:
Would be a tiny bit nicer with a when here, so that if we get a new entry in the hierarchy the IDE will give us a compile-time error here instead of just defaulting to else even though perhaps we'd want to do something else there.